pyproject.toml

```diff
@@ -1,21 +1,33 @@
 [build-system]
-requires = [
-  # sync with setup.py until we discard non-pep-517/518
-  "setuptools>=45.0",
-  "setuptools-scm[toml]>=6.2.3",
-  "wheel",
-]
-build-backend = "setuptools.build_meta"
-
-[tool.setuptools_scm]
-write_to = "pytest_metadata/__version.py"
+ requires = ["poetry-core>=1.0.0"]
+ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "pytest-metadata"
-version = "2.0.0"
+version = "2.0.1-rc0"
 description = "pytest plugin for test session metadata"
-authors = ["Dave Hunt"]
 license = "Mozilla Public License 2.0 (MPL 2.0)"
+authors = ["Dave Hunt <dhunt@mozilla.com>", "Jim Brännlund <jimbrannlund@fastmail.com>"]
+readme = "README.rst"
+homepage = "https://github.com/pytest-dev/pytest-metadata"
+repository = "https://github.com/pytest-dev/pytest-metadata"
+
+keywords = ["pytest", "metadata"]
+
+classifiers = [
+    "Topic :: Software Development :: Quality Assurance",
+    "Topic :: Software Development :: Testing",
+    "Topic :: Utilities",
+]
+
+packages = [
+    { include = "pytest_metadata", from = "src" },
+    { include = "ci", from = "src/pytest_metadata" },
```

Comment:
Is this required? I noticed you didn't have it in your PR @abn

Reply:
Looking in the built sdist tarball, `ci` is included.

Is it because it lives inside `src/pytest_metadata`?

Reply:
Do you want to exclude the package from builds? If so, use the exclude configuration.

You do not need the second package include since it gets auto detected as a package due to the `__init__.py`.

Reply:
No, I want to include it.

Ah, got it! Thanks (again 😅 )!